### ISSUE TITLE
Allow use HTML in the rules

### ISSUE THREAD

**Issue by RinCat** [mastodon/mastodon#19905]
### Pitch

Allow use HTML in the rules.

### Motivation

We had server rules in multi-language, and it's hard to read when compressed into a single line.
Attempts to use tags like `<br>` or `<hr>` have no effect.

**Comment by connyduck** [comment]
Even better would be a possibility to maintain the rules in different languages and users only see it in theirs.

**Comment by RinCat** [comment]
> Even better would be a possibility to maintain the rules in different languages and users only see it in theirs.

That would be better, but needs a lot of works. I prefer this temporary workaround to make it into 4.0 if possible.

**Comment by ClearlyClaire** [comment]
If I remember correctly, rules are purposefully kept in plain text so they can easily be used in many contexts, as in the report modal, apps, e-mails…

I do very much agree with the need to eventually support translating rules in multiple languages, though.

**Comment by RinCat** [comment]
> If I remember correctly, rules are purposefully kept in plain text so they can easily be used in many contexts, as in the report modal, apps, e-mails…
> 
> I do very much agree with the need to eventually support translating rules in multiple languages, though.

Yeah, I do remember it kept in plain text, but it should support use new line.

**Comment by ryliejamesthomas** [comment]
> If I remember correctly, rules are purposefully kept in plain text so they can easily be used in many contexts, as in the report modal, apps, e-mails…
> [...]

Previous issue: https://github.com/mastodon/mastodon/issues/16292

**Comment by RinCat** [comment]
> > If I remember correctly, rules are purposefully kept in plain text so they can easily be used in many contexts, as in the report modal, apps, e-mails…
> > [...]
> 
> Previous issue: #16292

OK, so I changed it to use new lines, that does not need to use HTML tags.
